Moolya.robot

```diff
@@ -0,0 +1,152 @@
+*** Settings ***
+Library          SeleniumLibrary
+Resource         C:\\Users\\Admin\\Documents\\GitHub\\Mentorship_Abhijit\\BrowserKeywords.robot
```

Comment:
- Use relative path for importing the files 
- Add newline before Suite Setup

Reply:
Yes sure